pkg/sql/logictest/testdata/logic_test/row_level_ttl

```diff
@@ -28,17 +28,145 @@ CREATE TABLE public.tbl (
                                                                                             FAMILY fam_0_id_text_crdb_internal_expiration (id, text, crdb_internal_expiration)
 ) WITH (ttl = 'on', ttl_automatic_column = 'on', ttl_expire_after = '00:10:00':::INTERVAL)
 
-statement error  resetting "ttl_expire_after" is not permitted\nHINT: use `RESET \(ttl_automatic_column\)` to remove the automatic column or use `RESET \(ttl\)` to remove TTL from the table
+statement error resetting "ttl_expire_after" is not permitted\nHINT: use `RESET \(ttl_automatic_column\)` to remove the automatic column or use `RESET \(ttl\)` to remove TTL from the table
 ALTER TABLE tbl RESET (ttl_expire_after)
 
+statement error cannot drop column crdb_internal_expiration while row-level TTL is active
+ALTER TABLE tbl DROP COLUMN crdb_internal_expiration
+
 query T
 SELECT reloptions FROM pg_class WHERE relname = 'tbl'
 ----
 {ttl_expire_after='00:10:00':::INTERVAL}
 
+query I
+SELECT count(1) FROM [SHOW SCHEDULES]
+WHERE label LIKE 'row-level-ttl-%'
+----
+1
+
+statement ok
+ALTER TABLE tbl SET (ttl_expire_after = '10 days')
+
+query T
+SELECT create_statement FROM [SHOW CREATE TABLE tbl]
+----
+CREATE TABLE public.tbl (
+                                                                                           id INT8 NOT NULL,
+                                                                                           text STRING NULL,
+                                                                                           crdb_internal_expiration TIMESTAMPTZ NOT VISIBLE NOT NULL DEFAULT current_timestamp():::TIMESTAMPTZ + '10 days':::INTERVAL ON UPDATE current_timestamp():::TIMESTAMPTZ + '10 days':::INTERVAL,
+                                                                                           CONSTRAINT tbl_pkey PRIMARY KEY (id ASC),
+                                                                                           FAMILY fam_0_id_text_crdb_internal_expiration (id, text, crdb_internal_expiration)
+) WITH (ttl = 'on', ttl_automatic_column = 'on', ttl_expire_after = '10 days':::INTERVAL)
+
+statement error cannot modify TTL whilst another schema change is in progress
```

Comment:
should we make this error clearer? It might be confusing because we're in an implicit transaction, also because it feels like it's all `ttl` related.

Reply:
i've gone with `cannot modify TTL settings while another schema change on the table is being processed`